packages/spacecat-shared-rum-api-client/README.md

```diff
@@ -298,6 +298,43 @@ Calculates the amount of non-inorganic (earned and owned) traffic and the click-
 
 ```
 
+### formVitals
+
+Checks the form vitals for a given domain within the requested interval. It checks whether the
+URL has forms embedded in it or not.
+
+An example response:
+
+```json
+[
+  {
+    "url": "https://business.adobe.com/",
+    "formsubmit": {},
+    "formview": {},
+    "formengagement": {
+      "desktop:windows": 100
+    },
+    "pageview": {
+      "desktop:mac": 800,
+      "desktop:windows": 1900,
+      "mobile:ios": 100,
+      "mobile:android": 300
+    }
+  },
+  {
+    "url": "https://business.adobe.com/se/resources/main.html",
+    "formsubmit": {},
```

Comment:
It'll be great, if you can add the example of `formsubmit` and `formview` objects as well

Reply:
done